src/status_im/ui/screens/currency_settings/styles.cljs

```diff
@@ -0,0 +1,23 @@
+(ns status-im.ui.screens.currency-settings.styles
+  (:require-macros [status-im.utils.styles :refer [defstyle]])
+  (:require [status-im.ui.components.colors :as colors]))
+
+(def wrapper
+  {:flex             1
+   :background-color :white})
+
+(defstyle currency-item
+  {:flex-direction     :row
+   :justify-content    :space-between
+   :background-color   :white
+   :align-items        :center
+   :padding-horizontal 16
+   :ios                {:height 64}
+   :android            {:height 56}})
+
+(defstyle currency-name-text
+  {:color   colors/black
+   :ios     {:font-size   17
+             :letter-g    -0.2
```

Comment:
The alignment is fine now, but seriously, what is this `letter-g` property? :)

Reply:
`letter-spacing`  but a short version without `spacin` :)

Reply:
@psdp could you fix this typo ?

Reply:
@flexsurfer oh sorry, fixed now.